am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/models/AttributeData.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.amlib.internal.models;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+
+import java.util.Set;
+
+@Data
+@Builder
+@AllArgsConstructor
+public final class AttributeData {
```

Comment:
I'd create sub package called `uk.gov.hmcts.reform.amlib.internal.models.query` to group all query objects.